backend/de.metas.business/src/main/java/de/metas/bpartner/composite/SalesRepContact.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.bpartner.composite;
+
+import de.metas.user.UserId;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+import javax.annotation.Nullable;
+
+@Value
+@Builder
+public class SalesRepContact
+{
+	@NonNull UserId id;
+	@Nullable String value;
+	@Nullable String phone;
+	@Nullable String email;
+	@Nullable String lastName;
+	@Nullable String firstName;
```

Comment:
is it right that besides id everything is nullable? what kind of SalesRepContact could be that?